README.md

```diff
@@ -28,6 +28,20 @@ If CMake is not already available on your system, it can be obtained from https:
 If you are using an older system such as RHEL 7 or CentOS 7,
 an updated version of CMake is also available via the EPEL repository via the `cmake3` package.
 
+Example installation:
+
+```
+mkdir build
+cd build
+cmake ..
+sudo make install
+```
+
+
+### Example Usage
+
+See `example.cpp` for example usage of the L-BFGS optimizer.
```

Comment:
Maybe add a link to the example and briefly describe the setting?

```suggestion
See [`example.cpp`](example.cpp) for example usage of the L-BFGS optimizer in a linear regression setting.
```

Reply:
thanks :)